pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -464,57 +462,24 @@ impl CustomScan for PdbScan {
             builder.custom_state().sort_field = builder.custom_private().sort_field();
             builder.custom_state().sort_direction = builder.custom_private().sort_direction();
 
-            // store our query quals into our custom state too
-            builder.custom_state().quals = Some(
-                builder
-                    .custom_private()
-                    .quals()
-                    .expect("should have a Qual structure"),
-            );
-            builder.custom_state().segment_count = builder.custom_private().segment_count();
-
-            // now build up the var attribute name lookup map
-            unsafe fn populate_var_attname_lookup(
-                lookup: &mut HashMap<(i32, pg_sys::AttrNumber), String>,
-                iter: impl Iterator<Item = *mut pg_sys::Node>,
-            ) -> Option<()> {
-                let mut iter = iter.peekable();
-                while let Some(node) = iter.next() {
-                    let (varno, varattno, attname) = {
-                        let varno = node.as_int()?;
-                        let varattno = iter.next()?.as_int()?;
-                        let attname = iter.next()?.as_c_str()?.as_ptr();
-
-                        (varno, varattno, attname)
-                    };
-
-                    lookup.insert(
-                        (varno as _, varattno as _),
-                        CStr::from_ptr(attname).to_string_lossy().to_string(),
-                    );
-                }
+            // store our query into our custom state too
+            // TODO: Clone? Take?
```

Comment:
Thoughts welcome, here and below.

Reply:
What's the question?

Reply:
Trying to decide between `clone()`ing from the `PrivateData` vs `take()`ing. We were previously `take()`ing a field, which I think answers my question. But I could imagine that being confusing if we ever tried to access the field again.

Reply:
probably `.take()` because it's not needed in the privdata anymore and there's probably a teeny bit more overhead for the Clone.  Just my thoughts behind why it previously did `.take()`